lib/cisco_node_utils/cmd_ref/radius_server.yaml

```diff
@@ -9,20 +10,27 @@ accounting:
   default_value: true
 
 acct-port:
-  kind: int
-  get_command: "show running-config radius all"
+  multiple: true
```

Comment:
I think you could add this under the `ios_xr` header (since Nexus *doesn't* support multiple ports for the same host). Something like:

```yaml
acct-port:
   get_value: '/^radius-server host %s.* acct-port (\d+).*$/'
   set_value: '<state> radius-server host <ip> acct-port <port>'
   default_value: 1813
   nexus:
     kind: int
   ios_xr:
     # IOS XR can have multiple ports attached to a single host
     multiple: true
```

This way the Nexus getter logic will work as-is (without needing to de-array the value)